<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>734</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#configdialog{
	background: rgb(47, 47, 47);
	border-radius: 5px;
	border: 4px solid rgb(34, 98, 108)
}

QPushButton {
	background-color: rgb(30, 30, 30);
	border: 1px solid rgb(85, 150, 255);
	color: rgb(85, 150, 255);
	padding: 4px 10px;
	font: 75 10pt &quot;Segoe UI&quot;;
	font-weight: 500;
padding: 5px 25px;
}
QPushButton::hover {
	background-color: rgb(85, 150, 255);
	border: 1px solid rgb(30, 30, 30);
	font: 75 10pt &quot;Segoe UI&quot;;
	color: rgb(30, 30, 30);
	border-radius:5px;
	font-weight: 500;
}

QPushButton::disabled {
	color:rgb(30, 47, 103);
	background-color:rgb(85, 106, 132);
	border: 1px solid rgb(7, 23, 59);
	border-radius: 10px;
	font-size: 14px;
}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="connectionContainer" native="true">
     <layout class="QVBoxLayout" name="connectionLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color:white;
	font-size:12px;
}

QLabel::disabled{
	color: rgb(171, 171, 171);
	font-size:12px;
}
QCheckBox {
	color:white;
	font-weight:500;
}

QCheckBox::disabled {
	color:rgb(171, 171, 171);
}
QPushButton {
	background-color: rgb(30, 30, 30);
	border: 1px solid rgb(85, 150, 255);
	color: rgb(85, 150, 255);
	padding: 4px 10px;
	font: 75 10pt &quot;Segoe UI&quot;;
	font-weight: 500;
}
QPushButton::hover {
	background-color: rgb(85, 150, 255);
	border: 1px solid rgb(30, 30, 30);
	font: 75 10pt &quot;Segoe UI&quot;;
	color: rgb(30, 30, 30);
	border-radius:5px;
	font-weight: 500;
}

QPushButton::disabled {
	color:rgb(30, 47, 103);
	background-color:rgb(85, 106, 132);
	border: 1px solid rgb(7, 23, 59);
	border-radius: 10px;
	font-size: 14px;
}

QLineEdit::disabled {
	background-color: rgb(181, 181, 181);
}
QLineEdit::enabled {
	font-size: 11px;
	padding: 5px 10px;
	border: 1px solid rgb(80, 80, 80);
	border-radius: 4px;
	background-color:  rgb(60,60,60);
	color: rgb(200, 200, 200);
}

QLineEdit::hover {
	border: 1px solid rgb(120, 120, 120);
	border-radius: 4px;
	background-color:  rgb(80,80,80);
}

QLineEdit::focus {
	border: 1px solid rgb(100, 100, 100);
	border-radius: 4px;
	background-color:  rgb(70,70,70);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8,6,6">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="deleteButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(29, 16, 16);
padding:3px 6px;</string>
        </property>
        <property name="text">
         <string>USUÅƒ</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color:white;
font-size: 14px;
font-weight: 500;</string>
        </property>
        <property name="text">
         <string>Nazwa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QLineEdit" name="nameEdit">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="styleSheet">
         <string notr="true">padding: 5px 30px;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
